docs/upgrade/upgrade-opensource/upgrade-guide-from-5.2-to-5.4/metric-update-5.2-to-5.4.rst

```diff
@@ -10,3 +10,158 @@ Scylla 5.4 Dashboards are available as part of the latest |mon_root|.
+       be forwarded to a replica.
+   * - scylla_storage_proxy_coordinator_speculative_data_reads
+     - Number of speculative data read requests that were sent.
+   * - scylla_storage_proxy_coordinator_speculative_digest_reads 
+     - Number of speculative digest read requests that were sent.
+   * - scylla_storage_proxy_coordinator_throttled_writes
+     - Number of throttled write requests.
+   * - scylla_storage_proxy_coordinator_write_rate_limited
+     - Number of write requests which were rejected by replicas because rate 
+       limit for the partition was reached.
+   * - scylla_storage_proxy_coordinator_write_timeouts
+     - Number of write request failed due to a timeout.
+   * - scylla_storage_proxy_coordinator_write_unavailable
+     - Number write requests failed due to an "unavailable" error.
+   * - scylla_storage_proxy_coordinator_writes_coordinator_outside_replica_set
+     - Number of CQL write requests which arrived to a non-replica and had to 
+       be forwarded to a replica.
+   * - scylla_storage_proxy_coordinator_writes_failed_due_to_too_many_in_flight_hints
+     - Number of CQL write requests which failed because the hinted handoff 
+       mechanism is overloaded and cannot store any more in-flight hints.
+   * - scylla_storage_proxy_replica_cas_dropped_prune
+     - How many times a coordinator did not perfom prune after cas.
+   * - scylla_storage_proxy_replica_cross_shard_ops
+     - Number of operations that crossed a shard boundary.
+   * - scylla_storage_proxy_replica_forwarded_mutations
+     - Number of mutations forwarded to other replica nodes.
+   * - scylla_storage_proxy_replica_forwarding_errors
+     - Number of errors during forwarding mutations to other replica nodes.
+   * - scylla_storage_proxy_replica_reads
+     - Number of remote data read requests this node received.
+   * - scylla_storage_proxy_replica_received_counter_updates
+     - Number of counter updates received by this node acting as an update leader.
+   * - scylla_storage_proxy_replica_received_mutations
+     - Number of mutations received by a replica node.
+   * - scylla_transport_cql_request_bytes
+     - Counts the total number of received bytes in CQL messages of a specific kind.
+   * - scylla_transport_cql_requests_count
+     - Counts the total number of CQL messages of a specific kind.
+   * - scylla_transport_cql_response_bytes
+     - Counts the total number of sent response bytes for CQL requests of a specific kind.
+
+
+The following metrics are replaced in ScyllaDB 5.4
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* scylla_transport_auth_responses
+* scylla_transport_batch_requests
+* scylla_transport_execute_requests
+* scylla_transport_options_requests
+* scylla_transport_prepare_requests
+* scylla_transport_query_requests
+* scylla_transport_register_requests
+* scylla_transport_startups
+
+The above metrics are replaced by the ``scylla_transport_cql_requests_count``
```

Comment:
Trying to think on how to make it clearer, maybe something on the line of: "The above metrics are grouped under the single metric scylla_transport_cql_requests_count with a ```kind``` label to diffrentiate between the metrics.

Reply:
An example would help